src/router/index.ts

```diff
@@ -1,14 +1,17 @@
 import Vue from "vue";
 import VueRouter from "vue-router";
-import Home from "../views/Home.vue";
+import LicitatiiView from "../views/LicitatiiView.vue";
 
 Vue.use(VueRouter);
 
 const routes = [
   {
     path: "/",
-    name: "Home",
-    component: Home
+    redirect: "/licitatii"
```

Comment:
@azirbrooks For now the simplest solution here was to reroute `/` directly to `/licitatii`